### DISCUSSION TITLE
stats not show when running multiple nuclei scans

### DISCUSSION THREAD

**Discussion by AaronChen0** [projectdiscovery/nuclei#3969] [Q&A] [answered]
### Nuclei version:
v2.9.9 

### Current Behavior:
When running a nuclei scan with the following command, it works as expected showing stats every 30 seconds.
```
nuclei -t scan.yaml -bs 150 -l urls.txt -stats -si 30 -o output.txt
```
Then at the same time, I want to do a nuclei scan on other targets with the following commands. Nuclei is scanning but stats don't show up.
```
mkdir differentDir
cd differentDir
nuclei -t /path/to/scan.yaml -bs 150 -l /path/to/urls2.txt -stats -si 30 -o output.txt
```

### Expected Behavior:
Running multiple nuclei scans at the same time, stats can show up for each scan.

**Comment by MetzinAround** [comment]
Probably a silly question, but are you running the scan in a different terminal window? how do you make the second scan happen while the first is going?

**Reply by AaronChen0** [reply]
Yes.  I run the second scan in a different terminal window while the first is going.

**Comment by ehsandeep** [comment, marked as answer]
@AaronChen0 nuclei is not designed in to run multiple instances on the same machine at the same time, you can still run it, but it will have unexpected side effects, including this and possibly more!

**Reply by RaJiska** [reply]
@ehsandeep Would you be able to share why this is happening, and what kind of side effects shall be expected, as well as why there is this limitation? Thanks :)

**Reply by xqd-ai** [reply]
I tried to run multiple instances using Docker and it resolved this problem, do you recommend this solution to bypass these limitations?!
